ledger/src/blockstore.rs

```diff
@@ -1644,6 +1644,41 @@ impl Blockstore {
         self.transaction_status_cf.put(index, status)
     }
 
+    // Returns a transaction status if it was processed in a root, as well as a loop counter for
+    // unit testing
+    fn get_transaction_status_with_counter(
+        &self,
+        signature: Signature,
+    ) -> Result<(Option<(Slot, TransactionStatusMeta)>, u64)> {
+        let mut counter = 0;
+        for i in 0..=1 {
```

Comment:
it's not obvious to me why we need to loop twice here or what `i` represents.   Perhaps a better variable name and/or comment